templates/core/terraform/processor_function/variables.tf

```diff
@@ -12,4 +12,5 @@ variable "docker_registry_username" {}
 variable "docker_registry_password" {}
 variable "docker_registry_server" {}
 variable "servicebus_connection_string" {}
-variable "workspacequeue" {}
\ No newline at end of file
+variable "workspacequeue" {}
+variable "service_bus_deployment_status_update_queue" {}
```

Comment:
Nit: Unify naming (prefix): `servicebus` vs nothing vs `service_bus`. My suggestion is to prefix all of these with `service_bus`. Need to be careful to update the naming change everywhere.

Reply:
unified all names to common pattern